README.md

```diff
@@ -33,20 +33,21 @@ This project is a pure rust implementation of a quadcopter.
 ### Motors
 - Crazepony Emax Mt2213 935kv Brushless Motor
 
-## Installation
+## Installation (Pi)
 1) Install Rust https://www.rust-lang.org/en-US/install.html
 2) Install Protobuf 3. For the Pi, you may need to install from source since there are no ARM releases.
```

Comment:
We don't use protobuf anymore (at least now). Also we have that install_deps....sh script which should take care of 1 and 2.